docs/pvcscale.md

```diff
@@ -0,0 +1,197 @@
+# PVCScale Workload
+
+The PVCScale workload playbook is `workloads/pvcscale.yml` and will run the pvcscale workload on your cluster.
+
+PVCScale workload is designed to stress storage backend and provide information
+how fast PODS are stared depending on storage backend they require PVCs to be
```

Comment:
s/stared/started

Reply:
I think this would be clearer:

PVCScale workload stresses storage backend and determines how fast pods are started depending on storage backend. Each pod requires a PVC to be allocated and attached.

Reply:
fixed!